code/modules/battlemonsters/items/card.dm

```diff
@@ -0,0 +1,175 @@
+/obj/item/battle_monsters/card
+	name = "battle monsters card"
+	desc = "A battle monster's card."
+
+	var/card_type = BATTLE_MONSTERS_CARDTYPE_MONSTER
+	var/datum/battle_monsters/element/prefix_datum
+	var/datum/battle_monsters/monster/root_datum
+	var/datum/battle_monsters/title/suffix_datum
+
+	var/datum/battle_monsters/spell/spell_datum
+	var/datum/battle_monsters/trap/trap_datum
+
+	w_class = ITEMSIZE_TINY
+
+	//Card information here
+
+/obj/item/battle_monsters/card/New(var/turf/loc,var/prefix,var/root,var/title,var/trap,var/spell)
```

Comment:
Why does this override even exist?

Reply:
I thought it solved a bug because initialize wasn't working without it.